### ISSUE TITLE
[Wasm] Improve persistence synchronization 

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno#1707]
##  What would you like to be added:

Storing files in persisted folders are not properly synchronized with emscripten's backend.

## Why is this needed:
Storing files should be working.

## For which Platform:

- [ ] iOS
- [ ] Android
- [x] WebAssembly
- [ ] Windows

**Comment by MartinZikmund** [comment]
@jeromelaban still relevant?

**Comment by jeromelaban** [comment]
No, it was fixed by ensuring that we're initializing the storage properly.
